numpy/polynomial/_polybase.py

```diff
@@ -9,7 +9,7 @@
 import os
 import abc
 import numbers
-from typing import Callable
+from collections.abc import Callable
```

Comment:
I'm still stumped `Callable` **protocol** should nowadays be considered as an *abstract base class* of a *collection*, which it clearly isn't...

Reply:
Indeed. Perhaps [collections.abc](https://docs.python.org/3/library/collections.abc.html) is misnamed:
> This module provides [abstract base classes](https://docs.python.org/3/glossary.html#term-abstract-base-class) that can be used to test whether a class provides a particular interface; for example, whether it is [hashable](https://docs.python.org/3/glossary.html#term-hashable) or whether it is a [mapping](https://docs.python.org/3/glossary.html#term-mapping).